README.md

```diff
@@ -63,29 +37,20 @@ After you're finished please remove all the comments and instructions!
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> React.
-
-<!-- Features -->
+> React. 
 
 ### Key Features <a name="key-features"></a>
 - **[Display a list of books ..]**
 - **[Allows users to Add a book ..]**
 - **[Allows users to Remove a selected book..]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p><!-- LIVE DEMO -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
Good job with removing the link but as the previous reviewer requested, please remove the Live Demo section if you have no deployed project. You still have the heading.